builder/builder.go

```diff
@@ -88,6 +88,8 @@ type BuildOpt struct {
 	OnlyArtifact               *domain.Artifact
 	OnlyArtifactDestPath       string
 	EnableGatewayClientLogging bool
+	EarthlyVersion             string
```

Comment:
What if we combine these two fields into `BuiltinArgs variables.DefaultArgs`? That way we would be consistent with how we define them in the `BuildOpt` and `ConvertOpt` structs.
```suggestion
	BuiltinArgs                variables.DefaultArgs
```

Reply:
Thanks, added this